packages/main/src/internal/safeGetChildrenArray.ts

```diff
@@ -0,0 +1,4 @@
+import { Children, ReactNode } from 'react';
+
+export const safeGetChildrenArray = <T = Exclude<ReactNode, boolean | null | undefined>>(children: unknown): T[] =>
```

Comment:
Should we change this implementiation to:
```
  // Not using React.Children.toArray because it mutates the key prop.
  let childArray: ReactElement[] = [];
  React.Children.forEach(children, child => {
    if (React.isValidElement(child)) {
      childArray.push(child);
    }
  });
```
If yes, then we should also credit react-spectrum for that: https://github.com/adobe/react-spectrum/blob/dfe72ee076f7ca27728d0b6b4c1ef86e11191ece/packages/%40react-spectrum/breadcrumbs/src/Breadcrumbs.tsx#L41-L47
and add it to the reuse file